dialect/sql/builder.go

```diff
@@ -2171,6 +2271,16 @@ var ops = [...]string{
 	OpNotNull: "IS NOT NULL",
 }
 
+// A ConflictResolutionOp represents a possible action to take when an insert conflict occurrs.
+type ConflictResolutionOp int
```

Comment:
SQLite calls it ["conflict resolution"](https://sqlite.org/lang_conflict.html) and PostgreSQL calls it ["conflict actions"](https://www.postgresql.org/docs/11/sql-insert.html#SQL-ON-CONFLICT). Both are fine to me, but move these constants above the `InsertBuilder.OnConflict` method.

Reply:
Naming is hard, originally I called it `conflictOp` but that sounded too menacing. I'll clean up the ordering as suggested.